addons/hi-res-thumbnails/addon.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "High resolution thumbnails",
+  "description": "Replaces thumbnails across the website with higher resolution images. Useful for Retina displays and browsing at greater than 100% zoom, and scales up thumbnails that are undersized by default.",
```

Comment:
@towerofnix The description is weird. `Retina` sounds too much like an Apple only thing, maybe "high pixel density displays"? And the last sentence looks out of place, and I'm not sure what it tries to convey.

Reply:
Yeah, I'm all for "high pixel density displays" :P It's not Apple-exclusive, obviously, but Retina displays afaik are the most like, well-known instance of high DPI displays? I went with it for recognizability—but teaching users to recognize that their Retina display is of the more general (and accurate) class/description "high pixel density display" is worthwhile.

The latter part is about scaling up images that are by default loaded at a *lower* resolution than what they're displayed at, so that they match that display (render) size. So for example, if an avatar is loaded at 38x38 but is displayed at 45x45, it'd automatically replace the dimensions with the larger ones (and then multiply by the user-selected scaling factor).

I'm not sure how to convey that in a simple sentence though, or if it's necessarily a key part of the addon (despite the fact that it's useful even if you don't use the high DPI part of the addon at all). Feedback would be appreciated!

Reply:
How about `Useful for Retina (high pixel density) displays`?  That seems simple enough and it clarifies exactly what you mean by Retina.

Reply:
Yeah, that'd be good! Another option is `Useful for high pixel density (e.g. Retina) displays`. Same difference really, but it points out explicitly that the more general term is high pixel density.